Could not update locale file 'locales/cs.yml' due to it being malformed

### Expected behavior

idk

### Observed/Actual behavior

1 - "Cannot load plugins\BentoBox\locales\Chat\cs.yml
org.bukkit.configuration.InvalidConfigurationException: mapping values are not allowed here
 in 'reader', line 28, column 71:
     ... red> <dark_red>[name]</dark_red>: [message]""

2 -

### Steps/models to reproduce

delete locales files as the update description in addon stated

### BentoBox version

BentoBox v3.14.2

### Plugin list

not needed for this im on 1.21.10 purpur

### Other

FULL LOG 
[01:48:20 INFO]: [BentoBox] Enabling BentoBox v3.14.2
[01:48:20 INFO]: [BentoBox] Loading Settings from config.yml...
[01:48:20 INFO]: [BentoBox] Saving default panels...
[01:48:21 INFO]: [BentoBox] Loading addons...
[01:48:21 INFO]: [PluginRemapper] Remapping plugin 'plugins\BentoBox\addons\AOneBlock-1.23.0.jar'...
[01:48:21 INFO]: [PluginRemapper] Done remapping plugin 'plugins\BentoBox\addons\AOneBlock-1.23.0.jar' in 97ms.
[01:48:21 INFO]: [BentoBox-AOneBlock] Loading server plugin BentoBox-AOneBlock v1.23.0
[01:48:21 INFO]: [PluginRemapper] Remapping plugin 'plugins\BentoBox\addons\Bank-1.9.1.jar'...
[01:48:21 INFO]: [PluginRemapper] Done remapping plugin 'plugins\BentoBox\addons\Bank-1.9.1.jar' in 27ms.
[01:48:21 INFO]: [BentoBox-Bank] Loading server plugin BentoBox-Bank v1.9.1
[01:48:21 INFO]: [BentoBox-Biomes] Loading server plugin BentoBox-Biomes v2.2.1
[01:48:21 INFO]: [PluginRemapper] Remapping plugin 'plugins\BentoBox\addons\Border-4.8.3.jar'...
[01:48:21 INFO]: [PluginRemapper] Done remapping plugin 'plugins\BentoBox\addons\Border-4.8.3.jar' in 26ms.
[01:48:21 INFO]: [BentoBox-Border] Loading server plugin BentoBox-Border v4.8.3
[01:48:21 INFO]: [PluginRemapper] Remapping plugin 'plugins\BentoBox\addons\Challenges-1.6.0.jar'...
[01:48:21 INFO]: [PluginRemapper] Done remapping plugin 'plugins\BentoBox\addons\Challenges-1.6.0.jar' in 85ms.
[01:48:21 INFO]: [BentoBox-Challenges] Loading server plugin BentoBox-Challenges v1.6.0
[01:48:22 INFO]: [PluginRemapper] Remapping plugin 'plugins\BentoBox\addons\Chat-1.4.0.jar'...
[01:48:22 INFO]: [PluginRemapper] Done remapping plugin 'plugins\BentoBox\addons\Chat-1.4.0.jar' in 22ms.
[01:48:22 INFO]: [BentoBox-Chat] Loading server plugin BentoBox-Chat v1.4.0
[01:48:22 ERROR]: [BentoBox] Could not update locale file 'locales/cs.yml' due to it being malformed: mapping values are not allowed here
 in 'reader', line 28, column 71:
     ... red> <dark_red>[name]</dark_red>: [message]
                                         ^

[01:48:22 ERROR]: Cannot load plugins\BentoBox\locales\Chat\cs.yml
org.bukkit.configuration.InvalidConfigurationException: mapping values are not allowed here
 in 'reader', line 28, column 71:
     ... red> <dark_red>[name]</dark_red>: [message]
                                         ^

	at org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:112) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:160) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:128) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.configuration.file.YamlConfiguration.loadConfiguration(YamlConfiguration.java:310) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	at BentoBox-3.14.2.jar/world.bentobox.bentobox.managers.LocalesManager.loadLocalesFromFile(LocalesManager.java:304) ~[BentoBox-3.14.2.jar:?]
	at BentoBox-3.14.2.jar/world.bentobox.bentobox.managers.AddonsManager.initializeAddon(AddonsManager.java:239) ~[BentoBox-3.14.2.jar:?]
	at BentoBox-3.14.2.jar/world.bentobox.bentobox.managers.AddonsManager.loadAddon(AddonsManager.java:196) ~[BentoBox-3.14.2.jar:?]
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) ~[?:?]
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?]
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1024) ~[?:?]
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151) ~[?:?]
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174) ~[?:?]
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
	at BentoBox-3.14.2.jar/world.bentobox.bentobox.managers.AddonsManager.loadAddons(AddonsManager.java:155) ~[BentoBox-3.14.2.jar:?]
	at BentoBox-3.14.2.jar/world.bentobox.bentobox.BentoBox.onEnable(BentoBox.java:172) ~[BentoBox-3.14.2.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:279) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:653) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:610) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:358) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1272) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:388) ~[purpur-1.21.10.jar:1.21.10-2535-0a2dc04]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: org.yaml.snakeyaml.scanner.ScannerException: mapping values are not allowed here
 in 'reader', line 28, column 71:
     ... red> <dark_red>[name]</dark_red>: [message]
                                         ^

	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchValue(ScannerImpl.java:902) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchMoreTokens(ScannerImpl.java:391) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.scanner.ScannerImpl.checkToken(ScannerImpl.java:238) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:637) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:161) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.comments.CommentEventsCollector$1.peek(CommentEventsCollector.java:57) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.comments.CommentEventsCollector$1.peek(CommentEventsCollector.java:43) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.comments.CommentEventsCollector.collectEvents(CommentEventsCollector.java:136) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.comments.CommentEventsCollector.collectEvents(CommentEventsCollector.java:116) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeScalarNode(Composer.java:241) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:205) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.getNode(Composer.java:131) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:157) ~[snakeyaml-2.2.jar:?]
	at org.yaml.snakeyaml.Yaml.compose(Yaml.java:575) ~[snakeyaml-2.2.jar:?]
	at org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:105) ~[purpur-api-1.21.10-R0.1-SNAPSHOT.jar:?]
	... 27 more

--- a/src/main/resources/locales/cs.yml
+++ b/src/main/resources/locales/cs.yml
@@ -25,4 +25,4 @@ chat:
       description: "Přepnout šmírování chatů na všech ostrovech"
       spy-on: <green>Šmírování chatů na ostrovech zapnuto</green>
       spy-off: <red>Šmírování chatů na ostrovech vypnuto</red>
-      syntax: <red>[island-chat-spy]</red> <dark_red>[name]</dark_red>: [message]
+      syntax: "<red>[island-chat-spy]</red> <dark_red>[name]</dark_red>: [message]"

